README.md

```diff
@@ -33,20 +62,85 @@ In that case you will be asked before old containers are removed.
 
 E.g.
 
-`> ./start --non-interactive`
+```sh
+$ ./start --non-interactive
+```
+
+If you want to use your own set of salt states you can provide `SALT_DIR`
+environment variable to the start script, so that directory will be used as salt
+root. For example, you could set this variable to a local clone of
+[kubic-project/salt](https://github.com/kubic-project/salt).
+
+Your velum folder will be mounted on the velum container, so any change you do
+locally to velum will be seen inside the container. Once that the service is up,
+you should be able to see the velum service at `http://localhost:3000/` on your
+local machine.
+
+After any change that you have performed you can run any rails/rake task as if
+it was local, with the docker prefix, like:
+
+```sh
+$ docker exec -it $(docker ps | grep velum-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rspec"
+$ docker exec -it $(docker ps | grep velum-dashboard | awk '{print $1}') bash -c "RAILS_ENV=test rubocop".
+```
+
+### Spawning test workers
+
+You can spawn new test workers with
+our [terraform](https://github.com/kubic-project/terraform) setup. In there you
+will notice that there are quite some options, but bear in mind that in order
+to work with this repository you should:
+
+- Set the `DASHBOARD_HOST` environment variable with the IP of your host.
+- Set the `SKIP_DASHBOARD` to 1, since Velum already provides that.
+
+So, for example, if the IP of your host is `192.168.1.33`, then you should
+perform the following command on the `terraform` directory:
+
+```sh
+$ DASHBOARD_HOST=192.168.1.33 SKIP_DASHBOARD=1 FLAVOUR=opensuse MINIONS_SIZE=2 contrib/libvirt/k8s-libvirt.sh apply
```

Comment:
I wasn't sure, because of the addition of microos images ;) Since we default to opensuse, it's ok, but I just wanted to stress this env. variable.